### PR TITLE
Passthrough CORS allowed origins

### PR DESCRIPTION
**Pull request by backjo**
Addresses #243

### CODE REVIEW

File: receiver/opencensusreceiver/config.go [hunk 1]
```diff
@@ -32,6 +32,8 @@ type Config struct {
 	// TLSCredentials is a (cert_file, key_file) configuration.
 	TLSCredentials *tlsCredentials `mapstructure:"tls-credentials,omitempty"`
 
+	CorsOrigins []string `mapstructure:"corsOrigins"`
```
Comment (songy23) [inline receiver/opencensusreceiver/config.go]: Please add document and update readme about this config.

Comment (backjo) [reply receiver/opencensusreceiver/config.go]: 👍

Comment (backjo) [reply receiver/opencensusreceiver/config.go]: So there's actually documentation already there - I just removed the TODO saying it didn't work.

Comment (songy23) [reply receiver/opencensusreceiver/config.go]: GoDoc is still needed, please add.

File: receiver/opencensusreceiver/config.go [hunk 2]
```diff
@@ -32,6 +32,8 @@ type Config struct {
 	// TLSCredentials is a (cert_file, key_file) configuration.
 	TLSCredentials *tlsCredentials `mapstructure:"tls-credentials,omitempty"`
 
+	CorsOrigins []string `mapstructure:"cors-allowed-origins"`
```
Comment (pjanotti) [inline receiver/opencensusreceiver/config.go]: Please, refer to the original comment to describe this field on OpenCensus service:
```golang
	// The allowed CORS origins for HTTP/JSON requests the grpc-gateway adapter
	// for the OpenCensus receiver. See github.com/rs/cors
	// An empty list means that CORS is not enabled at all. A wildcard (*) can be
	// used to match any origin or one or more characters of an origin.
```

File: receiver/opencensusreceiver/config.go [hunk 3]
```diff
@@ -32,6 +32,12 @@ type Config struct {
 	// TLSCredentials is a (cert_file, key_file) configuration.
 	TLSCredentials *tlsCredentials `mapstructure:"tls-credentials,omitempty"`
 
+	// CorsOrigins are the allowed CORS origins for HTTP/JSON requests to grpc-gateway adapter
+	// for the OpenCensus receiver. See github.com/rs/cors
+	// An empty list means that CORS is not enabled at all. A wildcard (*) can be
+	// used to match any origin or one or more characters of an origin.
+	CorsOrigins []string `mapstructure:"cors-allowed-origins"`
+
```
Comment (ccaraman) [inline receiver/opencensusreceiver/config.go]: There is a todo at line 50 for cors-allowed-origin. Can you remove it please?

File: receiver/opencensusreceiver/testdata/config.yaml [hunk 4]
```diff
@@ -47,6 +47,9 @@ receivers:
     # This receiver is disabled and won't receive any data.
     disabled: true
 
+  opencensus/cors:
```
Comment (ccaraman) [inline receiver/opencensusreceiver/testdata/config.yaml]: Can you add a few lines about what this does in this file? Ie: that it will accept requests from a.test.com but not wwwtest.com for ex?

File: receiver/README.md [hunk 5]
```diff
@@ -68,9 +68,6 @@ The full list of settings exposed for this receiver are documented [here](https:
 with detailed sample configurations [here](https://github.com/open-telemetry/opentelemetry-service/blob/master/receiver/opencensusreceiver/testdata/config.yaml).
 
 ### Writing with HTTP/JSON 
-// TODO(ccaraman) The cors setting wasn't ported accidentally. In a follow up
```
Comment (ccaraman) [inline receiver/README.md]: The next line is a bit outdated, like referring to OpenCensus. Could you reword it to mention Otel and make it clearer that the receiver handles both HTTP and gRPC.

Comment (backjo) [reply receiver/README.md]: A bit unclear here - is calling it the OpenCensus receiver incorrect? My understanding was that it's still being referred to as the OpenCensus receiver inside of Otel

Comment (ccaraman) [reply receiver/README.md]: 🤦‍♀ apologies my coffee is fading and rereading it a few more times I see that I was thinking about it in the wrong context. 
Ignore this comment!

**Comment by pjanotti** [comment]
Thanks @backjo !
